docs/testing/testing-overview.md

```diff
@@ -73,56 +61,8 @@ They run daily on continuous integration (TeamCity), because they can use networ
-
-We use [Jest](https://facebook.github.io/jest/) testing tool to execute all test configurations located in Calypso repository. It's highly recommended to use Jest's very flexible [API](https://facebook.github.io/jest/docs/en/api.html) together with [expect matchers](https://facebook.github.io/jest/docs/en/expect.html) and [mock functions](https://facebook.github.io/jest/docs/en/mock-function-api.html).
-
-Historically we have been using [Mocha](https://mochajs.org/) with [Chai assertions](http://chaijs.com/) and [Sinon mocks](http://sinonjs.org/). We still support Chai and Sinon for backward compatibility reasons, but Jest equivalents should be used whenever new tests are added.
-
-End-to-end tests use [Playwright](https://playwright.dev/docs/intro) to interact with the browser, and Jest to write and drive the testing scripts.
-
-### How to run all tests?
+End-to-end tests (`e2e` for short) describes the automated functional tests that simulate user interaction with the system under test.
 
-Executing `yarn test` from the root folder will run all test suites.
-Behind the scenes we maintain 3 test configuration. This is because each of them (`client`, `server`, and `integration`) has their own requirements.
+A well-maintained suite of e2e tests, in conjunction with CI (continuous integration) pipelines will ensure that regressions to key user flows are caught as early as possible in the development cycle.
 
-### How to run a smaller subset of test files?
-
-We have a yarn run script for each tests type: `yarn run test-client`, `yarn run test-server`, `yarn run test-integration`.
-You can pass a filename, folder name or matching pattern to these scripts to narrow down number of executed tests.
-
-Example for client:
-
-```bash
-> # run test suite for a specific test file
-> yarn run test-client client/state/selectors/test/get-media.js
-> # run test suites for all files in a specific folder
-> yarn run test-server server/config
-> # run test suites for all files matching pattern
-> yarn run test-client client/*/domains
-```
-
-### How to run specified suite or test-case
-
-The exclusivity feature allows you to run only the specified suite or test-case by appending `.only()` to the function.
-It works with `describe` and `it` functions. More details in [Jest documentation](https://facebook.github.io/jest/docs/api.html).
-
-Using `only` is a little bit dangerous, as you may end up committing the `only`, which would cause the test suite to only run your test on the build server. So be sure to look for stray only calls when reviewing a test. We have [eslint rules](https://github.com/jest-community/eslint-plugin-jest) that should catch them for you.
-
-Example:
-
-```js
-describe.only( 'just run this suite', function () {
-	test( 'should run these tests', function () {
-		// your test
-	} );
-
-	test.only( 'should only run this one test', function () {
-		// just run this test if the only is present
-	} );
-} );
-```
```

Comment:
Could you explain a little more about why we should pull this content out? Do these commands not work anymore?

Reply:
I think what you're seeing is artifact of the in-between state the repo was in, when I tried to combine `test/e2e` into this directory.

Reply:
Looking at the context a bit more, it looks like my intention was to make the overview page more generic and move the specifics like this above into its own pages.

Reply:
Moved this whole FAQ section under its own article, `faq.md`.